.github/workflows/deploy.yml

```diff
@@ -4,13 +4,21 @@ on:
     types:
       - published
```

Comment:
Can you not lock down the `published` type to a branch?

Reply:
Unfortunately you cannot. The hacky if statement below is my solution until GitHub (hopefully) allows this. When specifying a branch GitHub ignores it and continues to run the action for all releases.